pkg/dpm/lister.go

```diff
@@ -1,9 +1,26 @@
 package dpm
 
-type DeviceMap map[string][]string
+// ResourceLastNamesList contains last names of discovered resources. This type is used by Discover
+// implementation to inform manager about changes in found resources, e.g. last name of resource
+// "color.example.com/red" is "red".
+type ResourceLastNamesList []string
```

Comment:
This is a weird name. Having something like DevicePluginList is nicer and more fitting to what this really is.

Reply:
Done